server/lists/adapters/case-view-all-stages.js

```diff
@@ -40,7 +40,12 @@ const getRequestStatus = (dueDate, { status, decision }) => {
 
 async function composeBusinessLabel(choices, { businessArea, businessUnit }, fromStaticList) {
     const businessUnitLabel = await loadValue(businessUnit, choices, fromStaticList);
-    return businessArea ? `${businessArea} - ${businessUnitLabel}` : businessUnitLabel;
+    if (businessArea && businessUnitLabel) {
+        return `${businessArea} - ${businessUnitLabel}`;
+    }
+    else {
+        return businessArea ? businessArea : businessUnitLabel;
+    }
```

Comment:
Could add a test for this logic. 

Give me a shout if you need a hand :)

Cheers,
Rob

Reply:
I've added a test, so both execution paths should be covered now.

Thanks,
Joseph